build/release.mk

```diff
@@ -64,8 +64,9 @@ endif
 	cp README.md $(DIST_PATH)
 
 	@# Download prepackaged plugins
+	mkdir -p tmpprepackaged
 	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
-		curl -s https://api.github.com/repos/mattermost/$$plugin_package/releases/latest | awk -F\" '/browser_download_url/ { system("cd $(DIST_PATH)/prepackaged_plugins; curl -OL " $$4) }';\
+		cd tmpprepackaged && curl -OL https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz; \
```

Comment:
small nit, if `-O -L ...` is an acceptable substitute, would be a little easier to understand